source/features/show-open-prs-of-forks.tsx

```diff
@@ -32,7 +32,7 @@ const countPRs = cache.function(async (forkedRepo: string): Promise<[prCount: nu
 	`);
 
 	// Only show PRs originated from the current repo
-	const prs = search.nodes.filter((pr: AnyObject) => pr.headRepository.nameWithOwner.toLowerCase() === getRepoURL());
+	const prs = search.nodes.filter((pr: AnyObject) => pr.headRepository.nameWithOwner === getRepositoryInfo()!.url);
```

Comment:
Since `getRepositoryInfo` normalizes it, we also don't need to use `toLowerCase` on the source.

Also this makes me think that I should use `.nameWithOwner` instead of `.url`, making it even longer 😅

```diff
- getRepoURL()
+ getRepositoryInfo()!.nameWithOwner
```

Reply:
If I remember correctly we had that and you did not like it. So you split it.... ( now I cant seem to find that PR 😠 )

Reply:
https://github.com/sindresorhus/refined-github/pull/3231/files

Reply:
Where does it say `nameWithOwner`?